commit e9ef08bdc189e98610bc4b9a6e6f19bc3793b2c8
Author: Fenghua Yu <fenghua.yu@intel.com>
Date:   Fri Dec 8 16:06:01 2006 -0800

    [IA64] Itanium MC Error Injection Tool: Kernel configuration
    
    This patch has kenrel configuration changes for the MC Error Injection
    Tool.
    
    Signed-off-by: Fenghua Yu <fenghua.yu@intel.com>
    Signed-off-by: Tony Luck <tony.luck@intel.com>

diff --git a/arch/ia64/defconfig b/arch/ia64/defconfig
index 9001b3fbaa32..327e1f511f8c 100644
--- a/arch/ia64/defconfig
+++ b/arch/ia64/defconfig
@@ -144,6 +144,7 @@ CONFIG_COMPAT=y
 CONFIG_IA64_MCA_RECOVERY=y
 CONFIG_PERFMON=y
 CONFIG_IA64_PALINFO=y
+# CONFIG_MC_ERR_INJECT is not set
 CONFIG_SGI_SN=y
 
 #